app/views/admin/admin_users/show.html.erb

```diff
@@ -56,7 +56,22 @@
 
   <p class="submit"><%= submit_tag ts("Update") %></p>
 <% end %>
-
 <%= render :partial => 'user_history', :locals => {:user => @user} %>
 <!--/content-->
 </div>
+<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.0/jquery.min.js" type="text/javascript"></script>
```

Comment:
We already load jQuery and jQuery UI on all our pages, so you'll need to remove lines 62-71. In order to ensure that the jquery.timeago.min.js script loads after jQuery, you _could_ do this:

```
<%= content_for :footer_js do %>
  <script src="/javascripts/jquery.timeago.min.js" type="text/javascript"></script>
  <script type="text/javascript">
    $(document).ready(function() {
      $("abbr.timeago").timeago();
   });
  </script>
<% end>
```
However, it's a pretty useful script that I can easily see us wanting to use elsewhere on the site, so it would be even better to add it directly to the /views/layouts/_javascripts.html.erb file: `<%= javascript_include_tag "jquery.scrollTo.min", "jquery.livequery.min", "rails", "application", "bootstrap/bootstrap-dropdown.min", "jquery-shuffle", "jquery.tokeninput.min", "jquery.trap.min", "ao3modal.min", "jquery.cookie", "jquery.timeago.min" %>`

Then you can put `$j('.timeago').timeago();` in the topmost section of /public/javascripts/application.js: 

```
$j(document).ready(function() {
    setupToggled();
    if ($j('#work-form')) { hideFormFields(); };
    hideHideMe();
    showShowMe();
    handlePopUps();
    attachCharacterCounters();
    setupAccordion();
    setupDropdown();

    // remove final comma from comma lists in older browsers
    $j('.commas li:last-child').addClass('last');

    // make Share buttons on works and own bookmarks visible
    $j('.actions').children('.share').removeClass('hidden');

    // convert an absolute time to a relative time, e.g. 1 day ago
    $j('.timeago').timeago();

    prepareDeleteLinks();
    thermometer();
});
```

(You don't need to use `abbr.timeago` because we'll never use the class name `timeago` without intending to call this script.)